onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -808,6 +814,21 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.args) != 2,
         reason="arange_start_step overload takes three arguments (input, start, step)",
     ),
+    skip(
+        "atleast_1d_single_tensor",
+        matcher=lambda sample: isinstance(sample.input, (list, tuple)),
```

Comment:
So we don’t take a Sequence as input? Where is this op used? How are the inputs produced?

Reply:
It accepts both tensor and list.

Reply:
Oh I see there are two variants